scripts/feature_tests/upgrade/upgrade.sh

```diff
@@ -28,42 +28,46 @@ source "${METAL3_DEV_ENV_DIR}/lib/images.sh"
 # source <script name>.sh <log file prefix>
 # -----------------------------------------
 
-# Run cluster level ugprade tests
-pushd "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade" || exit
-# shellcheck disable=SC1091
-source 1cp_1w_bootDiskImage_cluster_upgrade.sh
-popd || exit
+if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
+    # Run cluster level ugprade tests
+    pushd "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade" || exit
+    # shellcheck disable=SC1091
+    source 1cp_1w_bootDiskImage_cluster_upgrade.sh
+    popd || exit
 
-pushd "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade" || exit
-# shellcheck disable=SC1091
-source clean_setup_env.sh
-popd || exit
+    pushd "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade" || exit
```

Comment:
This is repeated as in L33, can't we stay within the path instead of popd in L36, and popd only after we execute  clean_setup_env.sh in L40?

Reply:
shall be handled. Not before merge because the CI is so difficult

Reply:
fixed in https://github.com/metal3-io/metal3-dev-env/pull/497